### PR TITLE
Aws rds Integration support

### PR DESCRIPTION
**Pull request by YANG-DB**
### Description
add support for Integration (observability schema) on aws rds logs - using the pre-canned dashboards

### Issues Resolved
- https://github.com/opensearch-project/dashboards-observability/issues/695

### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/schemas/logs_vpc-1.0.0.mapping.json [hunk 1]
```diff
@@ -5,6 +5,9 @@
   "priority": 900,
   "data_stream": {},
   "template": {
+    "aliases": {
+      "logs-aws-vpc": {}
+    },
```
Comment (derek-ho) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/schemas/logs_vpc-1.0.0.mapping.json:10]: why this alias?

File: server/adaptors/integrations/repository/integration.ts [hunk 2]
```diff
@@ -211,7 +211,7 @@ export class Integration {
       );
       try {
         const ndjson = await fs.readFile(sobjPath, { encoding: 'utf-8' });
-        const asJson = '[' + ndjson.replace(/\n/g, ',') + ']';
+        const asJson = '[' + ndjson.trim().replace(/\n/g, ',') + ']';
```
Comment (derek-ho) [inline server/adaptors/integrations/repository/integration.ts:214]: separate PR please

File: server/adaptors/integrations/__data__/repository/aws_rds/schemas/aws_rds-1.0.0.mapping.json [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+                  "type": "double"
+                },
+                "sq-rows-sent": {
+                  "type": "long"
+                },
+                "sq-rows-examined": {
+                  "type": "long"
+                },
+                "sq-table-name": {
+                  "type": "text",
```
Comment (derek-ho) [inline server/adaptors/integrations/__data__/repository/aws_rds/schemas/aws_rds-1.0.0.mapping.json:76]: wondering here why these are mapped as text as opposed to keyword, any reason for doing so?

File: server/adaptors/integrations/__data__/repository/aws_rds/info/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+* Automating time-consuming administration tasks including hardware provisioning, database setup, patching, and backups
+
+RDS keeps your database up-to-date with the latest patches, and it also provides automatic backups and disaster recovery capabilities. You can make database instances available in multiple regions to enhance availability and reliability for your data.
+
+See additional details [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.html).
+
+## What is AWS RDS Integration?
+An integration is a collection of predefined assets which are combined in a meaningful manner.
+
+AWS RDS integration includes dashboards, visualizations, queries, and index mapping to help you manage and monitor your database services more effectively.
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_rds/info/README.md:19]: I think these READMEs would benefit a lot from a brief discussion of how to set up the service for locally testing the integration. People trying to integrate with RDS know what RDS is, but don't necessarily know how they get from point A to point B.

**Review by Swiddis — CHANGES_REQUESTED** [review]

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/schemas/logs_vpc-1.0.0.mapping.json (hunk 1 above)
Comment (YANG-DB) [reply server/adaptors/integrations/__data__/repository/aws_vpc_flow/schemas/logs_vpc-1.0.0.mapping.json:10]: used for simple search within the dashboard

File: server/adaptors/integrations/__data__/repository/aws_rds/aws_rds-1.0.0.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "aws_rds",
+  "version": "1.0.0",
+  "displayName": "AWS RDS",
+  "description": "AWS RDS",
+  "license": "Apache-2.0",
+  "type": "logs",
```
[the remark this answers is not in the document]
Comment (YANG-DB) [reply server/adaptors/integrations/__data__/repository/aws_rds/aws_rds-1.0.0.json:7]: type identified the Observability type as stated in the schema : Logs / Traces / Metrics

**Review by Swiddis — CHANGES_REQUESTED** [review]

**Review by anirudha — APPROVED** [review]

**Review by derek-ho — APPROVED** [review]

**Pull request merged**